tutorpod_autoscaling/plugin.py

```diff
@@ -41,48 +41,48 @@
     "lms": {
         "enable_hpa": True,
         "memory_request": f"{LMS_MEMORY_REQUEST_MB}Mi",
-        "cpu_request": 0.25,
+        "cpu_request": 0.5,
         "memory_limit": f"{LMS_MEMORY_REQUEST_MB * 4}Mi",
         "cpu_limit": 1,
         "min_replicas": 1,
         "max_replicas": LMS_MAX_REPLICAS,
-        "avg_cpu": 300,
+        "avg_cpu": 100,
         "avg_memory": "",
         "enable_vpa": False,
     },
     "lms-worker": {
         "enable_hpa": True,
         "memory_request": f"{LMS_WORKER_MEMORY_REQUEST_MB}Mi",
-        "cpu_request": 0.175,
+        "cpu_request": 0.25,
```

Comment:
Based on the previous scale numbers, I would increase the CPU request for LMS and CMS workers to 0.5 as well.

Reply:
Done